.github/workflows/pull_request_verification.yml

```diff
@@ -21,7 +21,13 @@ jobs:
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
       - name: Check lint
-        run: ./gradlew lintDebug
+        run: |
```

Comment:
shouldn't this be extracted to another step ? is there a reason to keep this together with "Check lint" step ?

Reply:
Yes, I can do this 😄